### PR TITLE
OTT-1397: Fix a bug that it did not correctly parse db_type and db_version for aurora mysql

### PR DESCRIPTION
**Pull request by YuzeLiao**

### CODE REVIEW

**Review by RobbieMcKinstry — APPROVED** [review]
LGTM but I'll defer to the others.

File: driver/driver_config_builder.py [hunk 1]
```diff
@@ -241,14 +241,27 @@ def from_rds(self, db_instance_identifier) -> BaseDriverConfigBuilder:
...
     def _get_cloudwatch_metrics_file(self, db_instance_identifier):
+        # For aurora mysql 5.6:
+        #   db_version = 5_6_mysql_aurora_1_22_2
+        #   db_type = aurora
+        # For aurora mysql 5.7:
+        #   db_version = 5_7_mysql_aurora_2_10_1
+        #   db_type = aurora_mysql
+        # For aurora postgres
+        #   db_version = 10_x / 11_x / 12_x
+        #   db_type = aurora_postgresql
```
Comment (NealKaviratna) [inline driver/driver_config_builder.py:252]: maybe change to docstring?

**Review by NealKaviratna — APPROVED** [review]
We'll need to bump the release version in the `compute_service_client` file as well.

**Comment by NealKaviratna** [comment]
> LGTM but I'll defer to the others.

oh looks like we haven't cut a release yet for the latest version with changes that Bohan landed, can maybe roll together

File: driver/driver_config_builder.py (hunk 1 above)
Comment (nappelson) [reply driver/driver_config_builder.py:252]: +1

Comment (YuzeLiao) [reply driver/driver_config_builder.py]: done

**Review by nappelson — APPROVED** [review]

**Pull request merged**